src/components/CustomerDetails/InputPassportSection.test.tsx

```diff
@@ -19,6 +19,7 @@ describe("InputPassportSection", () => {
       <InputPassportSection
         scannerType={"NONE"}
         openCamera={openCamera}
+        idInput=""
```

Comment:
why is the idInput here an empty string?

Reply:
The initial value of idInput is empty string. Did not declare it as state for the test component since setIdInput is not used in this case.